.gitignore

```diff
@@ -19,41 +19,10 @@ atlassian-ide-plugin.xml
-distribution/docs/src/main/resources/content/_catalogFrameworks/ingest_request.png	
-distribution/docs/src/main/resources/content/_catalogFrameworks/product_retrieval_request.png
-distribution/docs/src/main/resources/content/_catalogFrameworks/query_request.png
-distribution/docs/src/main/resources/content/_coreConcepts/.asciidoctor/
-distribution/docs/src/main/resources/content/_coreConcepts/highly-available-setup-after-failover.png
-distribution/docs/src/main/resources/content/_coreConcepts/highly-available-setup.png
-distribution/docs/src/main/resources/content/_data/.asciidoctor/
-distribution/docs/src/main/resources/content/_data/catalog_architecture_data.png
-distribution/docs/src/main/resources/content/_devComponents/.asciidoctor/
-distribution/docs/src/main/resources/content/_devComponents/sources_architecture.png
-distribution/docs/src/main/resources/content/_devComponents/XmlInputTransformer_configuration.png
-distribution/docs/src/main/resources/content/_devComponents/XmlInputTransformer_saxeventhandlerdelegate.png
-distribution/docs/src/main/resources/content_devComponents/source_architecture.png
-distribution/docs/src/main/resources/content/_plugins/.asciidoctor/
-distribution/docs/src/main/resources/content/_plugins/create_plugin_order.png
-distribution/docs/src/main/resources/content/_plugins/delete_plugin_order.png
-distribution/docs/src/main/resources/content/_plugins/query_plugin_order.png
-distribution/docs/src/main/resources/content/_plugins/resource_plugin_order.png
-distribution/docs/src/main/resources/content/_plugins/storage_create_plugin_order.png
-distribution/docs/src/main/resources/content/_plugins/storage_update_plugin_order.png
-distribution/docs/src/main/resources/content/_plugins/update_plugin_order.png
-distribution/docs/src/main/resources/content/_securityFramework/.asciidoctor/
-distribution/docs/src/main/resources/content/_securityFramework/security_architecture.png
-distribution/docs/src/main/resources/content/_securityFramework/security_ecp_architecture.png
-distribution/docs/src/main/resources/content/_securityFramework/security_idp_architecture.png
-distribution/docs/src/main/resources/content/_sources/.asciidoctor/
-distribution/docs/src/main/resources/content/_sources/embedded-solr-diagram.png
-distribution/docs/src/main/resources/content/_sources/http-solr-external-diagram.png
-distribution/docs/src/main/resources/content/_sources/http-solr-internal-diagram.png
-distribution/docs/src/main/resources/content/_sources/solrcloud-diagram.png
-distribution/docs/src/main/resources/content/_transformers/.asciidoctor
-distribution/docs/src/main/resources/content/_transformers/catalog_architecture_transformers.png
-distribution/docs/src/main/resources/content/_transformers/transformer_communication_diagram.png
+distribution/docs/src/main/resources/content/*/*/.asciidoctor
+distribution/docs/src/main/resources/content/*/*/*.png
```

Comment:
Commenting on this file so I can start a thread.

✏️ I see 2 `managing-intro.adoc` files. One under `_introduction` and one under `_managing`.
✏️ Why is _developing split into _devComponents and _devGuidelines when the outline structure is 
```
Developing
+ 18. Catalog Framework API
+ 19. Transformers
+ 20. Catalog Plugins
21. Operations
+ 22. Resources
+ 23. Queries
+ 24. Metrics
+ 25. Action Framework
26. Asynchronous Processing Framework
+ 27. Migration API
+ 28. Security Framework
+ 29. Developing DDF Components
+ 30. Development Guidelines
```
✏️ `_reference` appears to contain all of the content for the appendices. Should `_metadataReference` be under `_reference`? Should `_tables` be moved from `_reference`?

Reply:
I guess I jumped the gun on following the new outline. I'll have PRs up soon to break up the developing sections into an `architecture` reference and a more task-oriented `developing section. The appendices are also getting separated out into a standalone `reference` guide and a metadata guide.